testing/utils.go

```diff
@@ -21,3 +25,40 @@ func ApplyValSetChanges(t *testing.T, valSet *tmtypes.ValidatorSet, valUpdates [
 
 	return newVals
 }
+
+// ABCIResponsesResultsHash returns a merkle hash of ABCI results
+func ABCIResponsesResultsHash(ar *tmprotostate.ABCIResponses) []byte {
+	return tmtypes.NewResults(ar.DeliverTxs).Hash()
+}
+
+// MakeCommit commits a block
```

Comment:
I welcome suggestions for better `godoc`.

Reply:
What about something like:

```
MakeCommit iterates through the provided validator set, creating a Precommit vote for each 
participant at the provided height and round. Each vote is signed and added to the VoteSet. 
Finally, the VoteSet is committed finalizing the block
```

Reply:
Did tendermint make these functions private?

Reply:
[Looks like](https://github.com/tendermint/tendermint/blob/master/types/test_util.go#L11)...